references/cli/def.go

```diff
@@ -449,30 +449,57 @@ func NewDefinitionGenDocCommand(c common.Args) *cobra.Command {
 		Short: "Generate documentation of definitions (Only Terraform typed definitions are supported)",
 		Long:  "Generate documentation of definitions",
 		Example: "1. Generate documentation for ComponentDefinition alibaba-vpc:\n" +
-			"> vela def doc-gen alibaba-vpc -n vela-system\n",
+			"> vela def doc-gen alibaba-vpc -n vela-system\n" +
+			"2. Generate documentation for local ComponentDefinition file alibaba-vpc:\n" +
+			"> vela def doc-gen alibaba-vpc --local alibaba-vpc.yaml\n",
```

Comment:
```suggestion
			"> vela def doc-gen alibaba-vpc.yaml\n",
```
Let's make it as simple as possible.